src/cli.ts

```diff
@@ -219,23 +219,23 @@ function transformMarkdown(buf: Buffer) {
         }
         break
       case 'js':
-        if (/^```$/.test(line)) {
+        if (/^(```|~~~)$/.test(line)) {
           output.push('')
           state = 'root'
         } else {
           output.push(line)
         }
         break
       case 'bash':
-        if (/^```$/.test(line)) {
+        if (/^(```|~~~)$/.test(line)) {
           output.push('`')
           state = 'root'
         } else {
           output.push(line)
         }
         break
       case 'other':
-        if (/^```$/.test(line)) {
+        if (/^(```|~~~)$/.test(line)) {
```

Comment:
Here the stored sequence (same length as opening) must be detected.

Reply:
@nichtich I applied it. Could you review that? I also register you as a co-author. Could you also sign cla/google?

Reply:
I did so with `voss@gbv.de`. Additional notes:

- Code could be simplified by doing `match = line.match(/^(```+|~~~+)(.*)$/)` once and reusing `match[1]` and `match[2]`
- Why `if (new RegExp(`^${sequence}$`).test(line))`  instead of  `line === sequence`?
- No tests cases yet?

Reply:
@nichtich I added extracting sequence code after checking the markdown starting point. Could you explain more about the first feedback? 

I reflected that  ```if (new RegExp(^${sequence}$).test(line)) instead of line === sequence?```! 

And I added a test case to ```docs/markdown.md```. Could I add more tests to this?

Thank you!